integration/tests/cook/test_cli.py

```diff
@@ -882,6 +882,10 @@ def __wait_for_progress_message(self, uuids):
         return util.wait_until(lambda: cli.show_json(uuids, self.cook_url)[1][0]['instances'][0],
                                lambda i: 'progress' in i and 'progress_message' in i)
 
+    def __wait_for_executor_completion_message(self, uuids):
```

Comment:
Should we put this in `util`?

Reply:
This function relies on the use of the `cli`. I didn't want to introduce dependency on the `cli` in the `util.py`.

Reply:
Got it